src/exporter/trace/stdout/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::exporter::trace;
+use std::any;
+use std::sync::Arc;
+
+#[derive(Debug)]
+/// Exporter
+pub struct Exporter {}
+
+#[derive(Debug)]
+/// Builder
+pub struct Builder {}
+
+impl Default for Builder {
+    /// Return the default Exporter Builder.
+    fn default() -> Self {
+        Builder {}
+    }
+}
+
+impl Builder {
+    /// Create a new exporter from the builder
+    pub fn init(self) -> Exporter {
+        Exporter {}
+    }
+}
+
+impl Exporter {
+    /// Create a new exporter builder.
+    pub fn builder() -> Builder {
+        Builder::default()
+    }
+}
+
+impl trace::SpanExporter for Exporter {
+    /// Export spans to stdout
+    fn export(&self, batch: Vec<Arc<trace::SpanData>>) -> trace::ExportResult {
+        println!("{:?}", batch);
```

Comment:
@knabben and others: thoughts on instead writing to a `Write`-r, rather than using `println!`? The `Builder` could then default to [`Stdout`](https://doc.rust-lang.org/std/io/struct.Stdout.html).

Reply:
Makes sense, I was thinking is use some JSON serialization as well, since now we have the dump of the object.